cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java

```diff
@@ -32,7 +39,7 @@
  */
 public class FlushToStorageCommand extends ContainerCommand {
 
-    private static final int REQUEST_TIMEOUT_SECONDS = 30;
+    private static final int REQUEST_TIMEOUT_SECONDS = 60 * 5;
```

Comment:
Is this intentional?

Reply:
While testing flushtostorage we faced issue while with 30 secs timeout.

Reply:
This is intentional, we wanted to increase the timeout for flushtostorage. While testing flushtostorage we faced issue with 30 secs timeout.